docs/sources/enterprise/access-control/permissions.md

```diff
@@ -118,3 +131,5 @@ The following list contains fine-grained access control scopes.
 | `settings:*`                                                                         | Restrict an action to a subset of settings. For example, `settings:*` matches all settings, `settings:auth.saml:*` matches all SAML settings, and `settings:auth.saml:enabled` matches the enable property on the SAML settings. |
 | `provisioners:*`                                                                     | Restrict an action to a set of provisioners. For example, `provisioners:*` matches any provisioner, and `provisioners:accesscontrol` matches the fine-grained access control [provisioner]({{< relref "./provisioning.md" >}}).  |
 | `datasources:*`<br>`datasources:id:*`<br>`datasources:uid:*`<br>`datasources:name:*` | Restrict an action to a set of data sources. For example, `datasources:*` matches any data source, and `datasources:name:postgres` matches the data source named `postgres`.                                                     |
+| `folders:*`<br>`folders:id:*`                                                        | Restrict an action to a set of folders. For example, `folders:*` matches any folder, and `folders:id:1` matches the folder whose ID is `1`.                                                                                      |
```

Comment:
nit: maybe worth mentioning that this scope can also be applied to dashboards. Something like `Restrict an action to a set of folders or dashboards within those folders.`